t/base/num.t

```diff
@@ -1,10 +1,17 @@
 #!./perl
 
+BEGIN {
+    chdir 't' if -d 't';
+    @INC = '../lib'; # needed to locate strict for instances of 'no warnings'
+}
+
```

Comment:
we could also disable warnings without the need of loading `warnings.pm` 
`BEGIN { ${^WARNING_BITS} = '' }`

Reply:
> we could also disable warnings without the need of loading `warnings.pm`
> `BEGIN { ${^WARNING_BITS} = '' }`

Having to use obscure internals such as `${^WARNING_BITS}` in a directory like `t/base` is a real code-smell.  IMO our test files should be readable by average Perl programmers and not rely upon such arcane features as that one in order to get things done.